src/components/deploymentConfig/DeploymentTemplateView/DeploymentConfigFormCTA.tsx

```diff
@@ -203,14 +235,9 @@ export default function DeploymentConfigFormCTA({
             {!isPublishedMode && (
                 <>
                     {isApprovalPending && state.latestDraft?.canApprove && !approveDisabled && ApproveRequestTippy ? (
-                        <ApproveRequestTippy
-                            draftId={state.latestDraft.draftId}
-                            draftVersionId={state.latestDraft.draftVersionId}
-                            resourceName="deployment template"
-                            reload={reload}
-                        >
+                        <ConditionalWrap condition={!showLockedDiffForApproval} wrap={approveRequestTippy}>
```

Comment:
Instead of this separate conditional wrap can we merge the above condition as well?

Reply:
done